components/switch/switch.js

```diff
@@ -9,6 +9,16 @@ import { SwitchMixin } from './switch-mixin.js';
  */
 class Switch extends SwitchMixin(LitElement) {
 
+	static get properties() {
+		return {
+			/**
+			 * REQUIRED: The text that is displayed for the switch label.
+			 * @type {string}
+			 */
+			text: { type: String, reflect: true }
```

Comment:
So since this is already in `SwitchMixin` you don't need to do this part, just add it to the top comment block in the file:
`@attr text - REQUIRED: The text that is displayed for the switch label.`

Reply:
Hmm I'm not sure if that works as well, or if it's fine how it is, because I noticed that with 
`@attr text - REQUIRED: The text that is displayed for the switch label.`, we get 

```
        {
          "name": "text",
          "description": "REQUIRED: The text that is displayed for the switch label."
        }
```
for the `attributes`, and nothing for the `properties`.

Reply:
Technically, the above still renders in the properties table as:

text |   | REQUIRED: The text that is displayed for the switch label. | --
-- | -- | -- | --

But in the above, it's missing it's type of `string` since that wasn't generated.

Reply:
I had also previously tried doing
`@property text - REQUIRED: The text that is displayed for the switch label. @type {string}`

but that seems to be the opposite of `@attr`, namely it generates
```
        {
          "name": "text",
          "description": "REQUIRED: The text that is displayed for the switch label."
        },
```
for the `properties`, and nothing for the `attributes`.

Reply:
Yup so in that case we can change it to `@attr {string} text - REQUIRED: The text that is displayed for the switch label.`. The analyzer generally works with standard jsdoc comment format, so there's pretty good documentation on that out there (e.g., https://jsdoc.app/tags-property.html).

If we did want it to appear in properties as well we could also add a `@property` line but I don't think we need that too since Daylight just needs the attributes.

Reply:
Ah, nice. Looks like that works!